src/pudl/package_data/eia860/page_map.csv

```diff
@@ -9,6 +9,8 @@ generator_retired,0,0,0,0,0,0,0,0,2,2,2,2,2,2,2,2,2,2,2,2,2,2
 generator,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 generator_solar_existing,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 generator_solar_retired,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1
+generator_wind_existing,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+generator_wind_retired,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1
```

Comment:
It seems odd to me that we have page numbers defined for all the early years when these tables (and presumably the solar tables) didn't exist.  Is this file just totally different from the file map, and skip footer/header maps?

Reply:
oh i meant to add -1s in here. it doesn't realllly matter bc these year/tabs will never be selected bc the backbone of possible year/tabs comes from the file_map. but i like it better when we are consistent. i will update them